build.gradle.kts

```diff
@@ -4,22 +4,25 @@ import com.jfrog.bintray.gradle.BintrayExtension
 import com.novoda.gradle.release.PublishExtension
 import org.jetbrains.kotlin.gradle.dsl.Coroutines
 import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 buildscript {
     repositories {
         mavenCentral()
         jcenter()
         google()
+        maven(url = "https://dl.bintray.com/kotlin/kotlin-dev")
         maven { setUrl("https://kotlin.bintray.com/kotlinx") }
+        maven { setUrl("http://dl.bintray.com/kotlin/kotlin-eap") }
```

Comment:
Probably all should use `url = `

Reply:
I have made this change everywhere in the code base